Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1014,6 +1014,23 @@ func _vjpElu<T: TensorFlowFloatingPoint>(
     return (y, { v in Raw.eluGrad(gradients: v, outputs: y) })
 }
 
+/// Computes `leakyRelu` of the specified tensor element-wise.
```

Comment:
- The API documentation here is a bit redundant in that it does not explain what `leakyRelu` is. Other documentation like `relu` is also unclear. It should be changed to using the standard terminology.
- Could you please make sure #160 is addressed in future PRs?

```suggestion
/// Returns a tensor by applying the leaky ReLU activation function to the specified tensor element-wise.
```

Reply:
@rxwei I'll start working on #160 for all the files, I just take the neighborhood functions as references and the documentation is pretty inconsistent so I make these mistakes. I'll work on #160 for math.swift this week.